src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,25 @@
+while (free W) {
+	move W
+}
+
+while (free N) {
+	move N
+}
+
+ while (free E) {
```

Comment:
Also, when Sophia and I were trying out an implementation like this, we were wondering if it was redundant to have to check if a direction is free and then move in that direction. For example, picobot wouldn't be able to move north if north wasn't free in the first place, so maybe it's unnecessary to have to define this?